### PR TITLE
telco5g: Make TEST_RUN_FEATURES variable available to be set in BM job configs

### PR DESCRIPTION
**Pull request by danimjaffe**
In order to run only specific features tests we would want to be able to change this variable for each BM job that we will run.

### CODE REVIEW

**Comment by danimjaffe** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-master-e2e-telco5g-cnftests

File: ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh [hunk 1]
```diff
@@ -242,7 +242,9 @@ else
     export FEATURES="${FEATURES:-sriov performance sctp xt_u32 ovn metallb multinetworkpolicy vrf bondcni tuningcni}"
 fi
 export VALIDATIONS_FEATURES="${VALIDATIONS_FEATURES:-$FEATURES}"
-export TEST_RUN_FEATURES="${TEST_RUN_FEATURES:-$FEATURES}"
+if [[ -z "${TEST_RUN_FEATURES:-}" ]]; then
```
Comment (Sandeepyadav93) [inline ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh:245]:
~~~
export TEST_RUN_FEATURES="${TEST_RUN_FEATURES:-$FEATURES}"
~~~
Previous is preffered syntax in bash and functionally equal.

**Comment by danimjaffe** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-telco5g

**Comment by danimjaffe** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-telco5g pull-ci-openshift-sriov-network-operator-master-e2e-telco5g-cnftests

**Comment by danimjaffe** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-master-e2e-telco5g-cnftests

**Comment by danimjaffe** [comment]
/pj-rehearse pull-ci-openshift-sriov-network-operator-master-e2e-telco5g-cnftests

**Comment by danimjaffe** [comment]
/pj-rehearse pull-ci-openshift-sriov-network-operator-master-e2e-telco5g-cnftests

**Comment by danimjaffe** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-cnftests

**Comment by Sandeepyadav93** [comment]
/lgtm
/pj-rehearse ack


only sriov test ran:-
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/48949/rehearse-48949-pull-ci-openshift-sriov-network-operator-master-e2e-telco5g-cnftests/1760577045936476160/artifacts/e2e-telco5g-cnftests/telco5g-cnf-tests/artifacts/test_results.html

**Pull request merged**